tests/unit/types/document/test_helper.py

```diff
@@ -0,0 +1,77 @@
+import types
+
+import pytest
+
+from jina import Document
+from jina.types.document.helper import DocGroundtruthPair
+
+@pytest.fixture(scope='function')
```

Comment:
scope is function by default right?

Reply:
yes..i like to be explicit so added function here

Reply:
like a lot the folder structure matching